docs/release_process.md

```diff
@@ -0,0 +1,319 @@
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2 tag: "2022.08.02"
+       checkout change-3
+       commit
+       commit
+       checkout main
+```
+
+### Create Release Notes on GitHub
+After you push the tag to GitHub, you should also
+[make a pre-release on GitHub][github-new-release] for the tag.
+
+1. Choose the tag you just pushed (e.g., `2022.08.02`)
+2. Type the same tag name for the releae title (e.g., `2022.08.02`)
+3. Click "Previous tag:" and choose the tag currently on production.
+   * You can find this at [the `__version__` endpoint][prod-version].
+4. Click the "Generate release notes" button!
+5. Check the pre-release box.
+6. Click "Publish release"
+
+
+## Release to Prod
+We leave the tag on [Stage][stage] for a week so that we (and especially QA)
+can check the tag on GCP infrastucture before we deploy it to production. To
+deploy the tag to production:
+
+1. File an [SRE ticket][sre-board] to deploy the tag to [Prod][prod].
+   * Include a link to the GitHub Release
+   * You can assign it directly to our primary SRE for the day
+2. When SRE starts the deploy, "cloudops-jenkins" will send status messages
+   into the #fx-monitor-engineering channel.
+3. When you see `PROMOTE PROD COMPLETE`, do some checks on prod:
+   * Check sentry prod project for a spike in any new issues
+   * Check grafana dashboard for any unexpected spike in ops
+   * Spot-check the site for basic functionality
+   * Ping SDET to run end-to-end tests on prod
+4. Update the GitHub Release from "pre-release" to a full release and reference the production deploy SRE Jira ticket.
+
+
+## Stage-fixes
+Ideally, every change can ride the regular weekly release "trains". But
+sometimes we need to make and release changes before the regularly scheduled
+release.
+
+### "Clean `main`" flow
+If a bug is caught on [Stage][stage] in a tag that is scheduled to go to
+[Prod][prod], we need to fix the bug before the scheduled prod deploy. If
+`main` is "clean" - i.e., nothing else has merged yet, we can use the regular
+GitHub Flow:
+
+1. Create a stage-fix branch from the tag. E.g.:
+   * `git branch stage-fix-2022.08.02 2022.08.02`
+   * `git switch stage-fix-2022.08.02`
+2. Make changes
+3. Create a pull request to `main`
+4. Address review comments
+5. Merge pull request
+6. Make and push a new tag. E.g.: `2022.08.02.01`
+
+```mermaid
+%%{init: { 'theme': 'base' } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       branch change-2
+       commit
+       checkout main
+       merge change-1
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2 tag: "2022.08.02"
+       checkout change-3
+       commit
+       commit
+       checkout main
+       branch stage-fix-2022.08.02
+       commit
+       checkout main
+       merge stage-fix-2022.08.02 tag: "2022.08.02.01"
+```
+
+### "Dirty `main`" flow
```

Comment:
Since this flow is fairly similar to the "clean `main`" flow, and will have the exact same end result if `main` is clean, _and_ given that it's easy to miss the requirement that `main` is clean, maybe we should consider making this the only flow? Would also make the docs shorter :)

Reply:
Oh, I guess the "Future" flow section below references it. That said, IMHO documentation that doesn't describe the _current_ state of things isn't super useful anyway, so I'd propose removing that section as well.

Reply:
I don't want to hold up this PR anymore :) But I agree we should either move to the future state or just document the current state, let's do that in a followup if you don't mind.